orte/mca/schizo/base/schizo_base_stubs.c

```diff
@@ -46,6 +46,11 @@ int orte_schizo_base_parse_env(char *personality,
     int rc;
     orte_schizo_base_active_module_t *mod;
 
+    if (NULL == personality) {
+        opal_output(0, "NULL PERSONALITY");
```

Comment:
Nah -- you can do it however you want.  😄  I only used "assert" to convey the spirit of what I was thinking this was.  @bosilca would probably hate us putting an `assert()` in the middle of the code.  😉 

But then again, if this case happens, will returning an error code up the stack make it exit cleanly?  Or will it just cause a segv later, anyway?